# Bepaling van warmtecapaciteit van een onbekend materiaal


# Introductie

Onbekende materialen kunnen geïdentificeerd worden door hun eigenschappen te meten. Een van deze eigenschappen is de warmtecapaciteit. In dit practicum gaan we de warmtecapaciteit van een onbekend materiaal bepalen door middel van een calorimeter experiment. Daarbij wordt een bepaalde massa van het materiaal naar een bekende temperatuur gebracht waarna het in een bekende hoeveelheid water met bekende temperatuur wordt geplaatst. Door de temperatuur van het water te meten na het mengen kan de warmtecapaciteit van het onbekende materiaal worden berekend.

# Theorie

De soortelijke warmte $c$ van een materiaal is gedefinieerd als de hoeveelheid warmte $Q$ die nodig is om de temperatuur $T$ van een kilogram van het materiaal met één graad Celsius (of één Kelvin) te verhogen:

$$
    c = \frac{Q}{m \Delta T}
$$ (eq:heat_capacity)

Waarbij $Q$ de hoeveelheid warmte in Joules is, $m$ de massa in kilogram is en $\Delta T$ de verandering in temperatuur is. Gegeven de wet van Black, die stelt dat de totale hoeveelheid warmte in een geïsoleerd systeem constant blijft, kunnen we de warmte die het onbekende materiaal verliest gelijkstellen aan de warmte die het water opneemt:

$$
    Q_{materiaal} = -Q_{water} 
$$ (eq:black)

wanneer we de massa's en de begintemperaturen van beide systemen kennen, maar slechts een van de twee soortelijke warmtes, kunnen we de onbekende soortelijke warmte berekenen. We combineren vergelijkingen [](#eq:heat_capacity) en [](#eq:black) om de volgende vergelijking te krijgen:

$$
    T_e = \frac{c_w m_w T_{w,b}+c_m m_m T_{m,b}}{c_w m_w + c_m m_m}
$$ (eq:combined)

Waarbij de subscripts $b$ en $e$ respectievelijk staan voor begintoestand en eindtoestand, $w$ voor water en $m$ voor het onbekende materiaal.

Bij metingen aan verschillende massa's van het onbekende materiaal en vervolgens een least square fit aan bovenstaande vergelijking kunnen we een precieze waarde voor de soortelijke warmte van het onbekende materiaal bepalen. Dat is, wanneer de warmtecapaciteit van bijvoorbeeld de beker te verwaarlozen is.


# Methode en materialen

## Ontwerp
De bovenstaande theorie wordt gebruikt om de soortelijke warmte van een onbekend materiaal te bepalen. Het experiment bestaat uit het verwarmen van verschillende massa's van het onbekende materiaal tot een bekende temperatuur, waarna het in een bekende hoeveelheid water met bekende temperatuur wordt geplaats. Door de temperatuur van het water te meten na het mengen kan de warmtecapaciteit van het onbekende materiaal worden berekend. Om de tijd voor het meten van meerdere materialen te reduceren, worden de data van de verschillende groepen in het lokaal samengevoegd. Van tevoren is afgesproken welke massa's door welke groep worden gemeten, en hoeveel water er gebruikt wordt.

## Materialen
Hieronder staat de lijst van benodigde materialen bij deze proef:
- Calorimeter
- Thermometer of temperatuursensor
- Verwarmingsbron 
- Diverse massablokjes van onbekend materiaal
- Weegschaal
- Water
- Maatcilinder of maatbeker


```{figure} figures/c_onbekend_setup.jpg
:width: 70%
:label: fig_c_onbekend_setup

Een schematische weergave van de opstelling
```


## Procedure
Bespreek wie welke massa's van het onbekende materiaal gaat meten.
Bespreek ook hoeveel water er gebruikt gaat worden.
Bepaal de begintemperaturen.
Hevel het aantal afgesproken massa's in de maatbeker. 
Roer voorzicht zodat de temperatuur homogeen is.
Noteer de hoogste gemeten temperatuur, dit is $T_e$.
Wissel de metingen uit met de andere groepen en voer de data-analyse uit.

```{note}
Hieronder staat een voorbeeld hoe je een grafiek moet opslaan en vervolgens in je document kunt oproepen.
Het voorbeeld kun je natuurlijk straks verwijderen.
```

# Resultaten


In [3]:
import numpy as np
import matplotlib.pyplot as plt

m_water = np.array([307.3, 218.0, 229.1, 137.5, 331.1]) / 1000  # gram naar kg
m_massa = 100 / 1000 
t_init_w = np.array([20.2, 20.3, 20.2, 21.2, 22.5])
t_init_m = np.array([69.0, 67.0, 69.0, 67.0, 67.0])
dT = np.array([1.6, 2.3, 2.6, 1.9, 0.7]) 
t_eind = t_init_w + dT
dtm = t_init_m - t_eind

cw = 4186  # J/(kg*K)

dQ_water = cw * m_water * (t_eind - t_init_w)

c_m_berekend = dQ_water / (m_massa * (t_init_m - t_eind))

gemiddelde_c = np.mean(c_m_berekend)
print(f"De gemiddelde soortelijke warmte is: {gemiddelde_c:.0f} J/(kg*K)")

De gemiddelde soortelijke warmte is: 384 J/(kg*K)


In [2]:
from scipy.optimize import curve_fit
def efit(dt,cw):
    return cw*0.1*dt
popt, pcov = curve_fit(efit,dtm,dQ_water)
print(popt,np.sqrt(pcov[0]))


[389.05929082] [62.44288102]


# Discussie en conclusie

Discussie

De gevonden waarde voor de soortelijke warmte, cm​=389 J/(kg·K), ligt zeer dicht bij de boekwaarden van een aantal bekende metalen.

Wanneer we deze waarde vergelijken met standaardwaarden, valt op dat koper een soortelijke warmte heeft van ongeveer 387 J/(kg·K)[1]. De afwijking van de meting ten opzichte van koper is minder dan 1%. Dit suggereert dat het onbekende materiaal zeer waarschijnlijk koper is, of een legering met vergelijkbare thermische eigenschappen zoals messing (±377 J/(kg·K)).

Analyse van de meetfouten
Een probleemn met de meetopstelling voor dit experiment was dat de opstelling niet geisoleerd was, waardoor warmte heeft kunnen ontsnappen naar de omgeving. Verder stond het raam naast onze opstelling open in verband met bepaalde onaangenamen geuren, terwijl te tempratuur buiten rond de -2 lag, dat zal er ook voor hebben gezorgt dat het water verder en sneller afkoelde,.

Conclusie

Het doel van dit experiment was het bepalen van de soortelijke warmte van een onbekend materiaal. Door middel van een  lineaire regressie met curve_fit is een waarde bepaald van 389 J/(kgK).

Op basis van deze resultaten kan geconcludeerd worden dat het onbekende materiaal koper is. 

[1] https://en.wikipedia.org/wiki/Copper